PublicationsRouter/v2/api/API.md

```diff
@@ -208,7 +296,7 @@ On **successful completion** of the request, the system will respond with 202 (A
     }
 
 
-## For Repositories
+## For Repositories and Publishers
```

Comment:
Is that true? (...and publishers).  Do they have access to all these endpoints? What will they see if they use them?